zScanner/ViewModels/Documents/DocumentsListViewModel.swift

```diff
@@ -27,8 +27,6 @@ class DocumentsListViewModel {
     init(database: Database, ikemNetworkManager: NetworkManager) {
         self.database = database
         self.networkManager = ikemNetworkManager
-        
-        loadDocuments()
```

Comment:
Why did you remove it?

Reply:
It was already called in view controller with updateDocuments method so it was called twice.

Reply:
I think there was some reason for it, but I don't remember it now. Let's see if it's gonna be ok :)